### PR TITLE
ES6ify common/views/svgs

### PR DESCRIPTION
**Pull request by philwills**
## What does this change?
ES6ifying typing etc.

## Does this affect other platforms - Amp, Apps, etc?
No

## Tested in CODE?
No, only locally

### CODE REVIEW

File: static/src/javascripts/projects/common/views/svgs.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    shareEmail,
+    shareFacebook,
+    sharePinterest,
+    shareGPlus,
+    externalLink,
+    tick,
+    notificationsOff,
+    notificationsOn,
+    glabsLogoSmall,
+    adblockCoinsUk: adblockCoins,
```
Comment (philwills) [inline static/src/javascripts/projects/common/views/svgs.js]: This mixture of directly listing some objects and using others as property/value pairs, melts my mind, but seems consistent with what was there before.

File: static/src/javascripts/projects/common/views/svg.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 // @flow
 export const addClassesAndTitle = (
     markup: string,
-    classes?: ?Array<string>,
-    title?: ?string
+    classes?: Array<string>,
```
Comment (philwills) [inline static/src/javascripts/projects/common/views/svg.js]: I couldn't find any nullable uses of this, only optional ones, so thought that worth making explicit.

File: static/src/javascripts/projects/common/views/svgs.js (hunk 1 above)
Comment (gustavpursche) [inline static/src/javascripts/projects/common/views/svgs.js]: If you change the import name for `adblockCoinsUS` (into `adblockCoinsUs`) and `membershipLogo` (into `logomembership`) there is no need for the mind-blowing mix anymore. :)

The `adblockCoinsUk` could be a separate variable declaration before that list ...

What do you think?

File: static/src/javascripts/projects/common/views/svgs.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
+// @flow
+/* global console */
```
Comment (gustavpursche) [inline static/src/javascripts/projects/common/views/svgs.js]: I don't see the need for this (anymore) ... Am I missing something?

Comment (philwills) [inline static/src/javascripts/projects/common/views/svgs.js]: Probably not, I didn't notice it.

File: static/src/javascripts/projects/common/views/svgs.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+// Include any images needed in templates here.
+// This file is only required by core, and so has a long cache time.
```
Comment (SiAdcock) [inline static/src/javascripts/projects/common/views/svgs.js]: This comment is now pretty meaningless too

**Review by SiAdcock — APPROVED** [review]
Looks good 👍

**Pull request merged**